charts/images.yaml

```diff
@@ -161,8 +161,8 @@ images:
   tag: "1.22.5"
 - name: node-problem-detector
   sourceRepository: github.com/kubernetes/node-problem-detector
```

Comment:
Nit:
```suggestion
  sourceRepository: github.com/gardener/node-problem-detector
```

Reply:
I left it so cause original source is Kubernetes.

Reply:
I think @timebertt has right and I would also vote to have it switched to `github.com/gardener/node-problem-detector`. Otherwise how would one know that this build comes from this fork?

/hold

Reply:
Thanks for applying the suggestion!

/hold cancel
/lgtm